xla/service/gpu/transforms/priority_fusion.cc

```diff
@@ -739,7 +796,10 @@ class PriorityFusionQueue {
   // avoid recomputing priorities multiple times before we dequeue a new
   // producer.
   absl::flat_hash_set<HloInstruction*> to_update_priority_;
-
+  absl::flat_hash_map<HloInstruction*, std::vector<HloInstruction*>>
+      operands_to_new_consumers_;
+  absl::flat_hash_map<HloInstruction*, GpuPerformanceModel::RunTimes>
+      operands_to_removed_consumers_runtimes;
```

Comment:
`operands_to_removed_consumers_runtimes_`

Class members should have a trailing underscore in the name: google.github.io/styleguide/cppguide.html#Variable_Names